views/event_details.hbs

```diff
@@ -46,3 +46,21 @@
   </div>
 
 </div>
+
+<script type="text/javascript">
+  function initMap() {
```

Comment:
Why is this inline but everything else in script files?

Reply:
so this is here for 2 reasons : 
1) if it was on frontend i can't pass the lat & lng for the map ( unless if i maybe do an api request )
2) can't be on backend cause of i'm manipulating the html

Reply:
Sure that'll do for me.